dev-libs/boehm_gc/boehm_gc-7.2d.recipe

```diff
@@ -7,8 +7,8 @@ deallocating memory that is no longer useful. The collector automatically \
 recycles memory when it determines that it can no longer be otherwise accessed."
 HOMEPAGE="http://www.hboehm.info/gc/"
 COPYRIGHT="1988, 1989 Hans-J. Boehm, Alan J. Demers
-	1991-1996 by Xerox Corporation.  All rights reserved.
-	1996-1999 by Silicon Graphics.  All rights reserved.
+	1991-1996 by Xerox Corporation. All rights reserved.
+	1996-1999 by Silicon Graphics. All rights reserved.
```

Comment:
Get rid of the "All rights reserved", that's technically part of the LICENSE. And the "by" also.